README.md

```diff
@@ -90,8 +129,26 @@ Place your JavaScript code inside of the provided `client/app` folder. Use modul
   global.HelloWorld = HelloWorldAppServer;
   ```
    
+### Rails View Helpers In-Depth
+Once the bundled files have been generated in your `app/assets/javascripts/generated` folder and you have exposed your components globally, you will want to run your code in your Rails views using the included helper method.
+
+This is how you actually render the React components you exposed to `window` inside of `clientGlobals` (and `global` inside of `serverGlobals` if you are server rendering).
+
+`react_component(component_name, props = {}, options = {})`
++ **react_component_name:** Can be a React component, created using a ES6 class, or `React.createClass`, or a generator function that returns a React component.
++ **props:** Ruby Hash which contains the properties to pass to the react object
```

Comment:
Ruby Hash which contains the properties to pass to the react object, or a JSON string. If you pass a string, we'll js escape it for you.